arduino/motor_module.h

```diff
@@ -101,10 +107,13 @@ class MotorModule : public arduinoio::UCModule {
       // steps (int32)
       // max_frequency (int32) = 1 / min_wait_
       char dir_pin = command[MOVE_LENGTH];
-      pulse_pin_ = command[MOVE_LENGTH + 1];
-      trigger_negative_pin_ = command[MOVE_LENGTH + 2];
-      trigger_positive_pin_ = command[MOVE_LENGTH + 3];
-      done_pin_ = command[MOVE_LENGTH + 4];
+      bool changed = Update(command[MOVE_LENGTH + 1], &pulse_pin_);
```

Comment:
if these members are initialized to something stable (-1?), initialized_ will be redundant.

inline initialization is clearer IMO, you could enable C++11 features by adding
target_compile_features( kamikaze PRIVATE cxx_constexpr cxx_decltype_auto )

to the CMakeLists.txt.